src/groupeddataframe/grouping.jl

```diff
@@ -901,29 +940,55 @@ colwise(fns::Union{AbstractVector, Tuple}, d::AbstractDataFrame) = [f(d[i]) for
 colwise(f, gd::GroupedDataFrame) = [colwise(f, g) for g in gd]
 
 """
-    by(d::AbstractDataFrame, keys, cols => f...; sort::Bool = false)
-    by(d::AbstractDataFrame, keys; (colname = cols => f)..., sort::Bool = false)
+    by(d::AbstractDataFrame, keys, cols => f; sort::Bool = false)
+    by(d::AbstractDataFrame, keys, [col1 => f1, col2 => f2], col3 => f3; sort::Bool = false)
+    by(d::AbstractDataFrame, keys, [:col1, :col2, :col3] .=> f; sort::Bool = false)
+    by(d::AbstractDataFrame, keys; sort::Bool = false, colname = cols => f...)
     by(d::AbstractDataFrame, keys, f; sort::Bool = false)
-    by(f, d::AbstractDataFrame, keys; sort::Bool = false)
+    by(f, keys, d::AbstractDataFrame)
 
 Split-apply-combine in one step: apply `f` to each grouping in `d`
 based on grouping columns `keys`, and return a `DataFrame`.
 
 `keys` can be either a single column index, or a vector thereof.
 
-If the last argument(s) consist(s) in one or more `cols => f` pair(s), or if
```

Comment:
Please also use the same text as the one I wrote for `combine`.

Reply:
Added the same text for those bullets.